packages/context/test/unit/binding-filter.unit.ts

```diff
@@ -25,7 +25,41 @@ describe('BindingFilter', () => {
       expect(filter(binding)).to.be.false();
     });
 
-    // TODO: filter by tag map, filter by regexp
+    it('accepts bindings MATCHING the provided tag regexp', () => {
+      const filter = filterByTag(/^c/);
+      binding.tag('controller');
+      expect(filter(binding)).to.be.true();
+    });
+
+    it('rejects bindings NOT MATCHING the provided tag regexp', () => {
+      const filter = filterByTag(/^c/);
+      binding.tag('dataSource');
+      expect(filter(binding)).to.be.false();
+    });
+
+    it('accepts bindings MATCHING the provided tag map', () => {
+      const filter = filterByTag({controller: 'my-controller'});
+      binding.tag({controller: 'my-controller'});
+      binding.tag({name: 'my-controller'});
+      expect(filter(binding)).to.be.true();
+    });
+
+    it('rejects bindings NOT MATCHING the provided tag map', () => {
+      const filter = filterByTag({controller: 'your-controller'});
+      binding.tag({controller: 'my-controller'});
+      binding.tag({name: 'my-controller'});
+      expect(filter(binding)).to.be.false();
+    });
+
+    it('rejects bindings NOT MATCHING the provided tag map with multiple tags', () => {
```

Comment:
Can we have a test that accepts bindings matching tag map with multiple tags?

Reply:
Added